src/components/page/www/page.jsx

```diff
@@ -13,13 +13,13 @@ var Page = React.createClass({
         return (
             <div className="page">
                 <div id="navigation" className={classes}>
-                    <Navigation />
+                    <Navigation/>
```

Comment:
I was under the impression the space in here was one of our style conventions. Thoughts @rschamp and @technoboy10 ?